src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
I would like to argue against the `FileStreamFactory::createFileStream(string $path, string $mode = "r");` (without a wall of text). 

If we include this, it would make this PSR far more complicated. We have to make sure to deal with file not found, handling open/closing files, modes, errors related to opening files etc. The interfaces would be less complex (and less exception interfaces) if we did not support this.

And as stated before, the user might just do something similar to: 

```php
$factory->createResourceStream(fopen($file, 'r'));
```